docs/api-docs.md

```diff
@@ -54,14 +54,7 @@ Initializes a new Josh, with options.
 
 **Example**  
 ```js
-const Josh = require("@joshdb/core");
-const provider = require("@joshdb/sqlite");
-
-// sqlite-based database, with default options
-const sqliteDB = new Josh({
-  name: 'mydatabase',
-  provider,
-});
+const Josh = require("@joshdb/core");
const provider = require("@joshdb/sqlite");

// sqlite-based database, with default options
const sqliteDB = new Josh({
  name: 'mydatabase',
  provider,
});
```

Comment:
wtf i don't know why that displays this shit, but this is line returns actually

Reply:
yea I don't know where the actual changes are haha